invenio_theme/assets/semantic-ui/less/invenio_theme/theme/globals/site.overrides

```diff
@@ -509,14 +509,22 @@ code {
   display: inline-block;
 }
 
+.display-inline {
```

Comment:
Is it ok to add all the various `!important`? Needed?

Reply:
yes, these are utility classes, to enforce a specific behaviour on demand. We allow the `important!` only for these.